cime_config/config_compsets.xml

```diff
@@ -36,6 +36,11 @@
 
 <!-- I single point forcing -->
 
+  <compset>                                                                                                
+    <alias>I1PtClm50Bgc</alias>                                                                            
+    <lname>2000_DATM%1PT_CLM50%BGC_SICE_SOCN_SROF_SGLC_SWAV</lname>                                        
+  </compset>             
+
```

Comment:
Using an I2000 case is fine.  I also wonder, however, how we should handle CO2 concentrations for production runs that are supposed to be simulating conditions in 2018 and beyond? @danicalombardozzi

Reply:
So do you have site observations of CO2? Or do you want to use the global observed historical CO2? If the former we should add those observations to the user-mod directory for the site. If the later we possibly need to use a historical compset rather than fixed 2000.

But, in either case I think it makes sense to start with this and maybe add a slightly different one when you have the CO2 data ready.

Reply:
No observations here, I think we should just use the global CO2 file used in the default model for 2018-present day.

Reply:
OK, you''ll probably want the final compset to be a historical transient one then. Something like...

HIST_DATM%1PT_CLM50%BGC_SICE_SOCN_SROF_SGLC_SWAV

And have it set the START date in the user-mods to 2018.  You'll also need to turn the landuse.timeseries file off (since landuse won't change). That should give you the nearest gridcell from the global: prescribed aerosols, Nitrogen-deposition, CO2, and population-density for 2018 onward.

It might take a little bit of fidgeting to get that to work the way you want it to.

Reply:
In talking with @wwieder we also thought this compset should point to the latest CLM physics version. So we'll keep adding a new compset as new physics options become available. Because some sites have crop, we also think that we should make this with Crop on, so that it's always covered.

Reply:
Note, also that currently for the NEON sites we aren't pointing to the NEON tower site atmospheric data. So we need to figure out how to loop that in. That could be done in datm or in the user-mod directory below. @jedwards4b

Reply:
OK, it's in a later version of CDEPS as @jedwards4b pointed out. If I point to the latest CDEPS the NEON tower site data is looped in, but the older version doesn't have it. So I'll try with a newer CDEPS version.

Reply:
@ekluzek I keep thinking about this in the context of how we want to set up a generic single point 1ptCLM case?  As I understand it a generic single point case will: 

- use HIST_DATM, where users can define the START and END dates for time time period of interest?  
- point to global GWSP3 DATM files?

I wonder if @danicalombardozzi, @negin513 and @swensosc have thought here?

Reply:
In order to bring this PR in I'm going to leave the simple compset in place. I've created an issue to detail the new compsets that we'll add.

#1363

Reply:
@wwieder if you want to point to GSWP2 DATM files you should use one of the standard compsets. These single point compsets come into play when you have tower site data available.